test/integration/targets/ec2_asg/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# defaults file for ec2_asg
+# Amazon Linux AMI 2017.09.0 (HVM), SSD Volume Type
+ec2_ami_image:
+  us-east-1: ami-8c1be5f6
+  us-east-2: ami-c5062ba0
```

Comment:
In a follow-up it would be nice to change this to use image_facts or similar to support more than us-east-1 and us-east-2.

(I tested in us-east-2)